asciidoc/modules/con_mg_changes-in-vertx-mqtt.adoc

```diff
@@ -21,3 +21,21 @@ The following example shows the use of `publish()` method in {VertX} {4} release
 client.publish("hello", Buffer.buffer("hello"), MqttQoS.EXACTLY_ONCE, false, false);
 client.publish("hello", Buffer.buffer("hello"), MqttQoS.AT_LEAST_ONCE, false, false);
 ----
+
+== Mqtt will message returns a buffer
+
+The `MqttWill` data object gets and sets a string message as a buffer instead of a byte array. XXX
```

Comment:
@vietj Does this sentence read correct?

Original note:
https://github.com/vert-x3/wiki/wiki/4.0.0-Deprecations-and-breaking-changes#%EF%B8%8F-removed-deprecated-methods

Reply:
I would rather say: _The `MqttWill` data object wraps a string message as a Vert.x buffer instead of a byte array._